ext/src/clicker.jl

```diff
@@ -0,0 +1,75 @@
+function DynamicalSystems.interactive_clicker(dds;
```

Comment:
is there a reason the variable is `dds` instead of the typical `ds` used in the codebase?